mods/species/vox/organs_vox.dm

```diff
@@ -200,25 +200,28 @@
 /obj/item/organ/internal/voxstack/proc/backup_inviable()
 	return 	(!istype(backup) || backup == owner.mind || (backup.current && backup.current.stat != DEAD))
 
-/obj/item/organ/internal/voxstack/replaced()
+/obj/item/organ/internal/voxstack/on_replacement()
 	if(!..()) return 0
 	if(prompting) // Don't spam the player with twenty dialogs because someone doesn't know what they're doing or panicking.
 		return 0
-	if(owner && !backup_inviable())
-		var/current_owner = owner
-		prompting = TRUE
-		var/response = alert(find_dead_player(ownerckey, 1), "Your neural backup has been placed into a new body. Do you wish to return to life as the mind of [backup.name]?", "Resleeving", "Yes", "No")
-		prompting = FALSE
-		if(src && response == "Yes" && owner == current_owner)
-			overwrite()
-	sleep(-1)
-	do_backup()
+	
+	//Need spawn here so that this interactive bit doesn't lock up init
+	spawn(0)
```

Comment:
Should be using a timer or a `waitfor = FALSE` proc.

Reply:
Yeah, I could probably get a timer done. I was just a bit lazy there.

Reply:
done!